### DISCUSSION TITLE
sending from wallet under Windows not working - not valid numeric amount - Chia 1.1.7

### DISCUSSION THREAD

**Discussion by srrsr** [Chia-Network/chia-blockchain#6586] [Support]
Whatever input in amount field, always there 

an error message .  

' Please enter a valid numeric amount '

Such bug makes all project nonsense . Basic thing in wallet not working . 

You may send cryptocurrency in but not out .

**Comment by XIII39** [comment]
Input 0.0 in the Fee field, unless you want to add a fee to the network then put 0.00001 or something - Just don't add 1!

**Comment by BrandtH22** [comment]
@srrsr , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
